sklearn/utils/multiclass.py

```diff
@@ -344,11 +344,33 @@ def _check_partial_fit_first_call(clf, classes=None):
 
         else:
             # This is the first call to partial_fit
+            if issparse(y):
+                raise ValueError("Sparse y is not supported.")
+
+            # Validate classes against unique labels in y
```

Comment:
Also apart from [this](https://github.com/scikit-learn/scikit-learn/pull/3907#issuecomment-73839329) comment, do you see anything that is YAGNI / unnecessary in the code?